ibm/mas_devops/roles/cp4d/defaults/main.yml

```diff
@@ -31,3 +31,14 @@ cpd_entitlement_key: "{{ lookup('env', 'CPD_ENTITLEMENT_KEY') | default(ibm_enti
 # for the Cloud Pak for Data operators.
 cpd_operators_namespace: "{{ lookup('env', 'CPD_OPERATORS_NAMESPACE') | default('ibm-cpd-operators', true) }}"
 cpd_instance_namespace: "{{ lookup('env', 'CPD_INSTANCE_NAMESPACE') | default('ibm-cpd', true) }}"
+cpd_release_version: "{{ lookup('env', 'CPD_RELEASE_VERSION') | default(lookup('env', 'CPD_PRODUCT_VERSION'), true) | default('4.0.9', true)  }}"
```

Comment:
I vote for simplifying variables here. Suggest to use cpd_release_version / CPD_RELEASE_VERSION in both cpd and cpd_service roles. We don't want users setting conflicting values here, it does not make sense for what we need in MAS.

@durera any comments here?

Reply:
removed all the `cpd_release_version` references and just kept the `cpd_product_version` thru the CP4D roles.